syntax/ast.mjs

```diff
@@ -192,6 +177,15 @@ export class Variant extends SyntaxNode {
     }
 }
 
+export class CallArguments extends SyntaxNode {
+    constructor(positional = [], named = []) {
+        super();
+        this.type = "CallArguments";
+        this.positional = positional;
+        this.named = named;
```

Comment:
`CallArguments` is a separate `SyntaxNode` for two reasons:

1. The tooling parser can store the span of `(...)` on it.
2. In `TermReference`, the `args` field is nullable. This allows us to distinguish between `-term` and `-term()`, which are functionally the same, but require different serialization.